command/expand.go

```diff
@@ -89,3 +90,15 @@ func expandSources(
 
 	return ch
 }
+
+// raw source returns the only object with given path.
+func rawSource(followSymlinks bool, srcurls ...*url.URL) <-chan *storage.Object {
```

Comment:
Let's use `rawSource`  to eliminate code duplication in  `expandSource`.

`expand.go` line 38
```go
return rawSource(followSymlinks, srcurl), nil
```

Reply:
changed to `return rawSource(followSymlinks, srcurl), nil` to avoid code duplication.

Reply:
Please revert `expand.go`, implementation is moved to `url`.